Makefile

```diff
@@ -62,6 +61,16 @@ libtwin.a_includes-y := \
 libtwin.a_files-$(CONFIG_LOGGING) += src/log.c
 libtwin.a_files-$(CONFIG_CURSOR) += src/cursor.c
 
+# Rendering
+ifeq ($(CONFIG_RENDERER_BUILTIN), y) ## Builtin
```

Comment:
The `ifeq` is not necessary. You can simply write as following:
```
libtwin.a_files-$(CONFIG_RENDERER_BUILTIN) += src/draw.c
```

Both `CONFIG_RENDERER_BUILTIN` and `CONFIG_RENDERER_PIXMAN` are mutually exclusive, meaning that it is not possible to build the two files at the same time. Thus, `ifeq` is redundant.